tarantool_test.go

```diff
@@ -807,7 +816,7 @@ func TestSchema(t *testing.T) {
 	ifield1 := index3.Fields[0]
 	ifield2 := index3.Fields[1]
 	if ifield1 == nil || ifield2 == nil {
-		t.Errorf("index field is nil")
+		t.Fatalf("index field is nil")
```

Comment:
"Replaced some calls of Errorf() with Fatalf() in tests. That change
prevents nil dereferences below in the code."

So it means it closes https://github.com/tarantool/go-tarantool/issues/150 ?

Reply:
Yes, that seems to close this issue.

Reply:
That's nice. Please, update PR description to match the commit one

Reply:
Done.